amplify/backend/function/postUser/src/index.py

```diff
@@ -10,27 +10,27 @@
 db = client.Client(f"wss://{endpoint}:{port}/gremlin", "g")
 
 def lambda_handler(event, context):
-    userData = event["userInfo"]
-    firstName = userData["firstName"]
-    lastName = userData["lastName"]
-    dob = userData["DOB"]
-    username = userData["username"]
+    userData = event["request"]["userAttributes"]
+    # Inputs currently required by cognito
+    firstName = userData["given_name"]
+    lastName = userData["family_name"]
+    username = userData["preferred_username"]
     email = userData["email"]
-    gender = userData["gender"]
-    bio = userData["bio"]
 
+    # Inputs not required by cognito
+    dob = ""
+    gender = ""
+    bio = ""
+
+    # Add a user
     query = f"g.addV('user').property('firstName', '{firstName}').property('lastName', '{lastName}').property('DOB', '{dob}').property('username', '{username}').property('email', '{email}').property('gender', '{gender}').property('bio', '{bio}');"
-        
-    try:
-        db.submit(query)
-        result = f"User : {username} created successfully"
-        statusCode = 201
-    except:
-        result = f"Server failed at creating user : {username}"
-        statusCode = 500
+    db.submit(query)
+    # Get the user's vertex ID
+    query = f"g.V().has('username', '{username}').id();"
+    vertexID = db.submit(query).all().result()
```

Comment:
I think the vertexId might be in the result object that is returned from `db.submit()`. Try logging `result` and `result.one()` to see what is stored in there. Might save us an additional query.
```suggestion
    result = db.submit(query)
    vertexId = result.one();
```